app/providers/cluster.tsx

```diff
@@ -61,7 +61,7 @@ function parseQuery(searchParams: ReadonlyURLSearchParams | null): Cluster {
             return Cluster.Testnet;
         case 'mainnet-beta':
         default:
-            return Cluster.MainnetBeta;
+            return DEFAULT_CLUSTER;
```

Comment:
I set the default network to devnet in #1, but it didn't display correctly for the initial load of the app without query params.
https://github.com/nitro-svm/explorer/blob/1b23b866955cba5826ea6776ffc24e7250054b7e/app/utils/cluster.ts#L70